src/compas_viewer/commands.py

```diff
@@ -460,3 +461,23 @@ def load_data():
 
 
 load_data_cmd = Command(title="Load Data", callback=lambda: print("load data"))
+
+# =============================================================================
+# =============================================================================
+# =============================================================================
+# Scene Form
+# =============================================================================
+# =============================================================================
+# =============================================================================
+
+
+def get_name(obj: "ViewerSceneObject"):
+    return str(obj.name)
+
+
+def get_show_box(obj: "ViewerSceneObject"):
+    """
+    'checkbox' to show(hide) the ViewerSceneObject.
+    'show' is a boolean attribute of ViewerSceneObject.
+    """
+    return "checkbox", "show"
```

Comment:
I think these functions should not be here. I don't feel these two is going to be called else where other than configuring the sceneform.